tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -1,146 +1,90 @@
 # Copyright (c) Microsoft Corporation and contributors. All rights reserved.
 # Licensed under the MIT License.
 
-parameters:
-- name: poolBuild
-  type: object
-  default: Large
-
-- name: testWorkspace
+# This template defines steps to setup the telemetry-generator tool so custom telemetry can be sent to Kusto.
+# Other yaml files can include it and then immediately run the telemetry-generator.
+# It's designed to be included by other yaml files inside a job section.
+
+# Path to the folder that contains the source code for the telemetry-generator package.
+# The default should work in most/all scenarios. Won't work if the repo is checked out as a "secondary repo" in some
+# pipeline (which we don't do today) because $(Build.SourcesDirectory) would then point to the root of the primary repo.
+- name: pathToTelemetryGenerator
   type: string
+  default: '$(Build.SourcesDirectory)/tools/telemetry-generator'
 
-# Custom steps specified by the "caller" of this template, to actually run perf tests
-- name: customSteps
-  type: stepList
-
-- name: loggerPackage
+# URL for the "Dev feed" in our ADO project. All Fluid packages will be resolved from here when installing the
+# dependencies for telemetry generator and aria-logger.
+- name: devFeedUrl
   type: string
-  default: '@ff-internal/aria-logger'
```

Comment:
Removed this as a parameter and hardcoded it in the template; we don't really have another logger for this scenario